### DISCUSSION TITLE
Not able to run forge build

### DISCUSSION THREAD

**Discussion by Aamirusmani1552** [foundry-rs/foundry#5535] [Help]
I am trying to compile my contracts using `forge build` but I am getting this error:
```bash
The application panicked (crashed).
Message:  failed to extract foundry config:
foundry config error: Unknown evm version: paris for setting `evm_version`

Location: config\src\lib.rs:448

This is a bug. Consider reporting it at https://github.com/foundry-rs/foundry

Backtrace omitted. Run with RUST_BACKTRACE=1 environment variable to display it.
Run with RUST_BACKTRACE=full to include source snippets.
```

I have installed necessary dependencies using `forge install`. But after running this build command, not able to run `forge install` again. Getting the same error. 
Need help

**Comment by pavankv241** [comment]
same here

**Reply by Aamirusmani1552** [reply]
Hey, i was getting this error because i had old version of foundry. Old version didn't had the paris support. Try updating to latest version and add `evm_version` paris in foundry config and it should work.

**Reply by pavankv241** [reply]
Now , that's solved but facing another issue , forge is taking too much time to compile the 80 files .. Is there any way to resolve  .. please

**Reply by Aamirusmani1552** [reply]
That is also happening with me actually. It is running slow after update. You can check if `via-ir` option is enable in foundry config. Remove it and you will see improvement. It is used for compiling files that are causing `stack too deep` error. If you get the error then you would have to enable the option. Maybe ask in other issue and if possible share the link here also.

**Reply by pavankv241** [reply]
Thanks for response.. I'll  look now

**Reply by confusedeinstein** [reply]
I'm facing the same error, can your please let me know to to set `evm_version` in config file my forge version is `forge 0.2.0 (e53c0d2 2023-02-05T00:09:49.051836Z)`
